app/controllers/hq_controller.rb

```diff
@@ -1,4 +1,15 @@
 class HqController < ApplicationController
+  NAVIGATION_BUTTONS= {
```

Comment:
Navigation order matters so should this not be an array of hashes rather than a hash of hashes?